source/storage/Cinder/using-nfs-for-cinder-with-rdo.html.md

```diff
@@ -10,20 +10,20 @@ wiki_last_updated: 2014-01-06
 
 *   Set the NFS share in Packstack:
 
-`   packstack --cinder-backend=nfs --cinder-nfs-mounts=`<IP/hostname>`.:/`<shared directory>
+        # packstack --cinder-backend=nfs --cinder-nfs-mounts=<IP/hostname>:/<shared directory>
 
-*   -   Create the directory on which the shares will be mounted:
+*   Create the directory on which the shares will be mounted:
 
-         mkdir -p /var/lib/cinder/mnt
+        # mkdir -p /var/lib/cinder/mnt
 
 *   Make the Cinder user the owner of the directory:
 
-        chown -v cinder.cinder /var/lib/cinder/mnt
+        # chown -v cinder.cinder /var/lib/cinder/mnt
 
 *   Configure the SELINUX on the compute nodes to allow access from the instances to the shares:
 
-         /usr/sbin/setsebool -P virt_use_nfs on
+        # /usr/sbin/setsebool -P virt_use_nfs on
 
 *   Restart all the Cinder related services:
 
-         for i in $(chkconfig --list | awk ' /cinder/ { print $1 } ' ); do service $i restart; done
+        # for i in $(systemctl list-unit-files | awk ' /cinder/ { print $1 } ' ); do systemctl restart $i; done
```

Comment:
I guess it would be better to explicitly list cinder services, it's not like there are 100s of them